src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileFields.tsx

```diff
@@ -257,7 +230,33 @@ const SettingsEditProfileFields: React.FC<SettingsEditProfileFieldsProps> = ({
                 <Flex alignItems="center">
                   <CheckCircleIcon fill="black60" mr={0.5} />
                   <Clickable
-                    onClick={handleIDVerification}
+                    onClick={async () => {
+                      try {
+                        await submitVerifyIDMutation({
+                          variables: { input: {} },
+                          rejectIf: res => {
+                            return res.sendIdentityVerificationEmail
+                              ?.confirmationOrError?.mutationError
+                          },
+                        })
+
+                        sendToast({
+                          variant: "success",
+                          message: `ID verification link sent to ${
+                            me?.email ?? ""
+                          }.`,
+                          ttl: 6000,
+                        })
+                      } catch (error) {
+                        logger.error(error)
```

Comment:
The previous implementation was setup to call [`captureException`](https://github.com/artsy/force/blob/7f0950971b7098e733231ff0172bf1a35af381a0/src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx#L39) directly.   

Adding `logger.error` to continue to send these errors to sentry. By my understanding, this should suffice but let me know if that isn't the case.

Reply:
Yup! That's what the logger is there for